### ISSUE TITLE
Error raised when running the engine cart

### ISSUE THREAD

**Issue by christinach** [geoblacklight/geoblacklight#866]
after bundle install
and bundle exec rake geoblacklight:server I get the error:
```
Bundler could not find compatible versions for gem "sprockets":
  In snapshot (Gemfile.lock):
    sprockets (= 4.0.0)

  In Gemfile:
    geoblacklight was resolved to 2.2.1, which depends on
      handlebars_assets was resolved to 0.23.7, which depends on
        sprockets (>= 2.0.0)

    sass-rails (~> 5.0) was resolved to 5.1.0, which depends on
      sprockets (>= 2.8, < 4.0)

    engine_cart (~> 2.0) was resolved to 2.3.0, which depends on
      rails was resolved to 5.2.4.1, which depends on
        sprockets-rails (>= 2.0.0) was resolved to 3.2.1, which depends on
          sprockets (>= 3.0.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by christinach** [comment]
I was using `ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-darwin18]`

**Comment by jrgriffiniii** [comment]
I can confirm that I've reproduced this after invoking `bundle exec rake geoblacklight:server` when using Ruby release 2.5.7. It should be noted that this error was *not* raised when I used Ruby release `2.4.4`.

**Comment by jrgriffiniii** [comment]
Rails 6.0.1 did fix this with https://github.com/rails/rails/issues/36954

**Comment by mejackreed** [comment]
We have fixed this in other Blacklight gems like this: https://github.com/projectblacklight/blacklight-gallery/blob/master/spec/test_app_templates/Gemfile.extra

**Comment by jrgriffiniii** [comment]
https://circleci.com/workflow-run/6396beca-04f0-4893-b924-67a17286d997 gets this resolved for Rails 5.1.z releases and Ruby version 2.5.z and 2.6.z. Builds for Ruby 2.7.0 are failing, but this appears to be a Blacklight upstream issue: https://circleci.com/gh/jrgriffiniii/geoblacklight/504

**Comment by jrgriffiniii** [comment]
https://github.com/projectblacklight/blacklight/pull/2226 resolved this for Ruby 2.7.0, we may just need to wait for the next patch or minor release for Blacklight.

**Comment by jrgriffiniii** [comment]
As this is no longer breaking the builds on CircleCI, I'm going to close this until it is reproduced.
